comp/metadata/internal/util/inventory_payload.go

```diff
@@ -147,6 +149,13 @@ func (i *InventoryPayload) collect(_ context.Context) time.Duration {
 	i.m.Lock()
 	defer i.m.Unlock()
 
+	// Collect is called every MinInterval second. To maintain the same order of request as we did in 7.50.0
+	// We need to warranty that metadata information gets sent to the backend at least 1 minute past the startup time.
+	// For more information please check https://app.datadoghq.com/notebook/7691993/postmortem-ir-25790-agent-7-51-increase-the-chances-of-delayed-hots-tags
```

Comment:
I'm not sure we should link to our private article in a open source repo. We might want to explain quickly what is the issue instead.